Cargo.toml

```diff
@@ -18,8 +18,11 @@ members = ["magnus-macros"]
 exclude = ["examples/rust_blank/ext/rust_blank", "examples/custom_exception_ruby/ext/ahriman", "examples/custom_exception_rust/ext/ahriman"]
 
 [features]
-embed = []
+embed = ["rb-sys?/link-ruby"]
```

Comment:
This roughly translates to: "activate the rb-sys/link-ruby feature *only if* rb-sys is already activated."

Reply:
This syntax was stabilised in Rust 1.60, and I was trying to keep Magnus working on older compiler versions (currently downto 1.51 is supported) so consumers of gems (who aren't necessarily Rust users) can use their distro's packaged Rust (which tends to be outdated).

Reply:
OK, I think we can stop using this feature when support for the built-in `ruby_sys` bindings is dropped.